README.md

```diff
@@ -1 +1,22 @@
-# one-click-installers
+# GleSYS one-click installers [![License: CC0-1.0](https://img.shields.io/badge/License-CC0%201.0-lightgrey.svg)](http://creativecommons.org/publicdomain/zero/1.0/)
+> The scripts that powers the one-click installers on GleSYS Cloud (cloud.glesys.com).
+
+## Guidelines
+
+1. The one-click installer should be placed in a folder which is named after a rundown of what it does. Eg. `wordpress-nginx-mariadb`.
+2. The folder should include a `cloudconfig.yaml`, which should include the cloud-init code. And a `README.md`, which should include a summary of what it does and the software that will be downloaded onto the server.
+3. Include other license in your `README.md` if the repo license isn't applicable for your contribution.
+4. When your cloud-init is merged, it still has to be integrated in the GleSYS Cloud interface. This doesn't happen automatically.
+
+
+## Contributing
+
+1. Fork it (<https://github.com/yourname/yourproject/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Commit your changes (`git commit -am 'Add some fooBar'`)
+4. Push to the branch (`git push origin feature/fooBar`)
+5. Create a new Pull Request
+
+## License
+
+GleSYS one-click installers is open-sourced software licensed under the [CC0 1.0 Universal](https://creativecommons.org/publicdomain/zero/1.0/deed.en). Separate one-click installers may use other licenses and if so, they should be visible in their `README.md`.
```

Comment:
It would be good if it was at all possible to have the license notice in the actual cloudinit file instead of in a readme alongside it. If the cloudinit file get distributed it's much more likely to retain the license notice in that case.

Other than that I am more than fine with this approach.

Reply:
Agree. It should suffice to include the license in the `cloudconfig.yaml` file. The `README.md` might be superfluous if the `cloudconfig.yaml` also can include a comment about what it does.

Reply:
The format for the documentation will change later down the road. I think it is nice to separate the documentation from the actual cloud-config, to make it easier to use in cloud.

I guess we could add license information as a comment to the cloud-config file. I'm fine with whatever.